sources/core/Stride.Core.Tests/TestSerialization.cs

```diff
@@ -89,6 +89,8 @@ public struct S
             public int A { get; set; }
         }
 
+        // TODO change binary serialization due to microsoft security concerns.
```

Comment:
Can you open a bug for this?

Reply:
https://github.com/stride3d/stride/issues/1596
added to Issues.